apps/docs/stories/components/checkout.stories.tsx.backup

```diff
@@ -1,6 +1,7 @@
 import type { Meta, StoryObj } from '@storybook/web-components';
```

Comment:
File extension on this is `.backup` so that it doesn't actually show in Storybook. Wanted to leave the file for now because it contains a simple approach to including a theme on the story